dump/parser.go

```diff
@@ -48,7 +49,10 @@ func Parse(r io.Reader, h ParseHandler, parseBinlogPos bool) error {
 			break
 		}
 
-		line = line[0 : len(line)-1]
+		// on windows someone end with '\r\n'
```

Comment:
The comment has some grammar mistakes. I prefer you can use:

```
Ignore '\n' on Linux or '\r\n' on Windows
```

Reply:
ok, I have change